src/app/modules/lti/pages/lti/lti.component.ts

```diff
@@ -36,7 +37,11 @@ const loginIdParam = 'login_id';
 export class LTIComponent implements OnDestroy {
 
   private loginId$ = this.activatedRoute.queryParamMap.pipe(
-    map(queryParams => queryParams.get(loginIdParam)),
+    map(queryParams => {
+      const loginId = Number(queryParams.get(loginIdParam) ?? NaN);
+      if (Number.isNaN(loginId)) throw new Error('login id must be a number');
+      return loginId;
+    }),
```

Comment:
Why converting it to a number? Because the service takes a number as an input? 
I am afraid this `integer <int64>` for input is just equivalent to `string <int64>` as query string are strings anyway... and the reason we use strings for ids is that JS does not have enough precision on numbers to support int64. As a consequence, by converting the string to number, you may loose precision while doing that, and so modify the id. :-/ 

So, tl;dr: use strings for ids in your service and here.

Reply:
fixed